single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -0,0 +1,373 @@
+
+##### Step 3: Import the sample notebooks
+
+After the Lakehouse is created, please click the 'New' button again, and select 'Import notebook' from the menu. Navigate to `src/notebooks` folder under the repo, and select all the notebooks to import:
+
+![import all notebooks](./images/import_all_notebooks.png)
+
+##### Step 4: Add the Created Lakehouse to the notebooks
+
+After importing, open the following notebooks and add the lakehouse created in the previous step to them:
+- data_cleansing
+- data_ingestion
+- data_transformation
+- data_validation
+- EDA
+- feature_set_registration
+
+Please note that the Lakehouse name should be '[prefix]_lh', where [prefix] is the prefix you defined in the prerequisites section.
+
+![add lakehouse](./images/add_lh_to_notebook.png)
+
+#### Create an Azure Resource Group
+
+Create a new resource group named as [prefix]rg, where [prefix] is the prefix you defined in the prerequisites section.
+
+#### Azure App Service
+
+Create a new Azure App Service named as [prefix]as, where [prefix] is the prefix you defined in the prerequisites section.
+
+#### Azure Data Lake Storage Gen2
+
+Create an Azure Data Lake Storage Gen2 account named as [prefix]sa, where [prefix] is the prefix you defined in the prerequisites section. For the detailed information, please refer to: [Create a storage account to use with Azure Data Lake Storage Gen2](https://learn.microsoft.com/en-us/azure/storage/blobs/create-data-lake-storage-account)
+
+#### Azure Purview
+
+Create an Azure Purview account named as [prefix]pv, where [prefix] is the prefix you defined in the prerequisites section.
+
+#### Azure Machine Learning Managed Feature store
+
+Create an Azure Machine Learning Managed Feature store by running the following Azure CLI:
+
+```az ml feature-store create --subscription <subscription id> --resource-group <resource group> --location <location> --name <feature store name>```
+
+For other ways to create a feature store, please refer to the online document:
+[Create a Minimal Feature Store](https://learn.microsoft.com/en-us/azure/machine-learning/tutorial-get-started-with-feature-store?view=azureml-api-2&tabs=SDK-track#create-a-minimal-feature-store).
+
+### Data Source Preparation
+
+In this demo the data from the [New York City Taxi & Limousine Commission](https://www.nyc.gov/site/tlc/about/tlc-trip-record-data.page) Yellow Taxi Trip Records, which is public. We used Records for 2022. This includes anonymized travel details like departure points, destinations, times, distances, and costs. The data, in conjunction with Taxi Zone Maps and Lookup Tables, aids in various research fields. It helps identify frequent pickup and drop-off zones in the city.
+
+We provide two options to get the sample data:
+
+- Option 1: ADLS (Azure Data Lake Storage) or Blob storage.
```

Comment:
For simplicity, I would recommend to just use Option 2 as default. We can mention option 1 in the footnote as an alternate approach without providing all the details.

Reply:
remove option 1 from the doc makes it more simpler and keep it as a footnote info. Thank you for your advice